### PR TITLE
HHH 13809 Fix some issues in Hibernate's user guide

### PR DESCRIPTION
**Pull request by NathanQingyangXu**
Tons of stuff. The issues fixed mainly focus on:

- grammatical errors
- code example format consistency (esp. the empty char delimiter usage)
- outdated references (e.g. missing 'basic type' mapping table entries)
- UI CSS tweaking (e.g. the '+', '-' buttons are too close; images are not positioned at centre, etc.)
- many more

### CODE REVIEW

File: documentation/src/test/java/org/hibernate/userguide/envers/EntityTypeChangeAuditTest.java [hunk 1]
```diff
@@ -110,11 +110,11 @@ public void test() {
...
-                    .get( entityManager )
-                    .getCrossTypeRevisionChangesReader()
-                    .findEntityTypes( 2 )
-                    .iterator().next()
-                    .getFirst()
+                        .get( entityManager )
+                        .getCrossTypeRevisionChangesReader()
+                        .findEntityTypes( 2 )
+                        .iterator().next()
+                        .getFirst()
```
Comment (Naros) [inline documentation/src/test/java/org/hibernate/userguide/envers/EntityTypeChangeAuditTest.java]: Your indentation used here seems inconsistent with that in all the other files modified in this commit.

Comment (NathanQingyangXu) [reply documentation/src/test/java/org/hibernate/userguide/envers/EntityTypeChangeAuditTest.java]: yeah, there are many other inconsistency issue. One question, should we replace 'tab' with empty chars to ensure predictable and consistent code snippet presenting. Currently I only ensure the indentation is correct on web, but mixing of tab and space chars seems not robust.

Comment (NathanQingyangXu) [reply documentation/src/test/java/org/hibernate/userguide/envers/EntityTypeChangeAuditTest.java]: I decided to close this PR temporarily. Especially I think I wanna exclude testing code format change from the scope of this PR (which has been big enough).

**Comment by NathanQingyangXu** [comment]
close the PR to better plan how to narrow the scope and make code review friendly

**Pull request closed without merging**